src/styles/blocks/section1.scss

```diff
@@ -0,0 +1,37 @@
+.section_1 {
+  margin: 0;
+  padding: 32px 16px;
+  height: 134px;
+  width: 166px;
+  text-align: center;
+}
+
+.section_1_img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 36px;
+}
+
+.section_1__text {
+  height: 36px;
+  width: 166px;
+  font-size: 12px;
+  font-weight: 500;
+  top: 175px;
+  margin: 0;
+  color: #060b35;
+  line-height: 18px;
+  font-family: Roboto, sans-serif;
+  text-align: left;
```

Comment:
The 'text-align' property is set to 'left', which is the default value for this property. If the intent is to override a previous setting, this is fine, but if not, you can remove this property to reduce redundancy.